src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/IViewHandler.Android.cs

```diff
@@ -7,6 +7,6 @@ public interface IAndroidViewHandler : IViewHandler
 	{
 		void SetContext(Context context);
 
-		AView View { get; }
+		AView? View { get; }
```

Comment:
everything would be so simple, or at least simpler, if the View was never null. could we consider this ? (View created as part of the ctor, e.g.)

Reply:
Agreed

What in here that's marked nullable can we guarantee won't be null.

Like does a ViewHandler with a null View make any sense?

Reply:
The views are created after the ctror and can be swapped out

https://github.com/xamarin/Xamarin.Forms/blob/7aedbc7d3e592a531879d2eb05a6daa673e16715/src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/View/AbstractViewHandler.cs#L43

I'm assuming we'll need this to be nullable